apps/test/unit/templates/lessonOverview/StudentLessonOverviewTest.jsx

```diff
@@ -184,4 +185,21 @@ describe('StudentLessonOverview', () => {
     const wrapper = shallow(<StudentLessonOverview {...newDefaultProps} />);
     assert.equal(wrapper.find('#unit-test-introduced-code').length, 0);
   });
+
+  it('displays levels with progress if levels present', () => {
+    const lessonLevels = fakeLevels(4);
+    const wrapper = shallow(
+      <StudentLessonOverview {...defaultProps} lessonLevels={lessonLevels} />
+    );
+    console.log(wrapper.find('#level-section'));
+    assert.equal(wrapper.find('#level-section').length, 1);
```

Comment:
I also see this often written as `expect(wrapper.find('#level-section')).to.have.length(1)`, but this works too!